src/Microsoft.PowerShell.ConsoleHost/WindowsTaskbarJumpList/TaskbarJumpList.cs

```diff
@@ -4,14 +4,55 @@
 using System;
 using System.Diagnostics;
 using System.Reflection;
+using System.Threading;
 
 using static Microsoft.PowerShell.ComInterfaces;
 
 namespace Microsoft.PowerShell
 {
     internal static class TaskbarJumpList
     {
-        internal static void CreateElevatedEntry(string title)
+        // Creating a JumpList entry takes around 55ms when the PowerShell process is interactive and
+        // owns the current window (otherwise it does a fast exit anyway). Since there is no 'GET' like API,
+        // we always have to execute this call because we do not know if it has been created yet.
+        // The JumpList does persist as long as the filepath of the executable does not change but there
+        // could be disruptions to it like e.g. the bi-annual Windows update, we decided to
+        // not over-optimize this and always create the JumpList as a non-blocking background task instead.
+        internal static void CreateRunAsAdministratorJumpList()
+        {
+            try
+            {
+                // Some COM APIs are STA only, therefore the executing thread must run in STA
+                if (Thread.CurrentThread.GetApartmentState() == ApartmentState.STA)
+                {
+                    TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator);
```

Comment:
I don't think you need to check for the current `ThreadApartment` because we don't want to run this in the current thread to slow down `pwsh` startup.
We should always start a new STA thread for this work.

Reply:
This code  is here thinking about the future when there will be an `-sta` switch on `pwsh` (issue #7216). Maybe call `Task.Run(() => TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator));` if the current thread is already STA (like before)?

Reply:
We don't want to run this logic on the current thread, so it doesn't matter if the current thread is STA or not. We shouldn't use thread pool thread because they will always be MTA.
So, here we should just start a new STA thread and run the jumplist logic.

Reply:
Thanks, you are right. I fixes this